### DISCUSSION TITLE
How to load the thinc model associated with a trained component ?

### DISCUSSION THREAD

**Discussion by opsomerto** [explosion/spaCy#9073] [Help: Other Questions] [answered]
Hello,

I'm wondering how can I load only the `model` from a trained textcat component ?

Context:
I have trained several textcat classifiers, and I would like to build a custom component that wraps several textcat classifier to implement a custom logic (apply some classifiers after other ones if some conditions are met...).

Thanks :)

**Comment by polm** [comment, marked as answer]
Main point in the spaCy source to check would be [`from_disk` in `TrainablePipe`](https://github.com/explosion/spaCy/blob/3b2575186fa18b7de47c0b22c7e4f91d3fa2ec0c/spacy/pipeline/trainable_pipe.pyx#L320). Basically this is just using the `Model.from_disk` function in Thinc, though you have to initialize the same model first so the structures are set up correctly.
